.circleci/config.yml

```diff
@@ -83,12 +85,11 @@ commands:
   deploy-pip-snapshot-unix:
     steps:
       - run: |
-          ulimit -n 100000
           export DEPLOY_PIP_USERNAME=$REPO_VATICLE_USERNAME
           export DEPLOY_PIP_PASSWORD=$REPO_VATICLE_PASSWORD
-          bazel run --define version=$(git rev-parse HEAD) //python:deploy-pip39 -- snapshot
-          bazel run --define version=$(git rev-parse HEAD) //python:deploy-pip310 -- snapshot
-          bazel run --define version=$(git rev-parse HEAD) //python:deploy-pip311 -- snapshot
+          bazel run --jobs=8 --define version=$(git rev-parse HEAD) //python:deploy-pip39 -- snapshot
+          bazel run --jobs=8 --define version=$(git rev-parse HEAD) //python:deploy-pip310 -- snapshot
+          bazel run --jobs=8 --define version=$(git rev-parse HEAD) //python:deploy-pip311 -- snapshot
```

Comment:
Bazel on Docker in CircleCI for whatever reason doesn't seem to limit the number of simultaneously running jobs